.github/workflows/dashboards-reports-test-and-build-workflow.yml

```diff
@@ -32,6 +32,7 @@ jobs:
 
       - name: Add Chromium Binary to Reporting for Testing
         run: |
+          sudo apt update
```

Comment:
I think this change is covered in Josh's PR  #245 , can you merge from upstream here?